.gitmodules

```diff
@@ -56,9 +56,9 @@
 [submodule "cmake/external/SafeInt/safeint"]
 	path = cmake/external/SafeInt/safeint
 	url = https://github.com/dcleblanc/SafeInt.git
-[submodule "cmake/external/onnx-tensorrt"]
-	path = cmake/external/onnx-tensorrt
-	url = https://github.com/onnx/onnx-tensorrt.git
 [submodule "cmake/external/optional-lite"]
 	path = cmake/external/optional-lite
 	url = https://github.com/martinmoene/optional-lite.git
+[submodule "cmake/external/onnx-tensorrt"]
```

Comment:
You need to update the https://github.com/microsoft/onnxruntime/blob/master/cgmanifests/submodules/cgmanifest.json
See https://github.com/microsoft/onnxruntime/blob/master/cgmanifests/README.md